### ISSUE TITLE
MapQuest tiles

### ISSUE THREAD

**Issue by daf** [NOAA-ORR-ERD/WebGnomeClient#7]
The Cesium (I think?) views are now just showing issues with MapQuest - I read somewhere they turned off free access to their tiles this week.

**Comment by naomiconnolly-noaa** [comment]
This is true, we'll be adding api keys or removing them as base layers options shortly.
